src/App.js

```diff
@@ -102,6 +103,9 @@ const App = () => {
                                 <GuardedRoute path='/create-post' meta={{ roles: [ROLE.USER, ROLE.AGENT] }}>
                                     <CreatePost/>
                                 </GuardedRoute>
+                                <Route exact path='/users/:username'>
```

Comment:
Add meta

Reply:
Meta is not necessary here because this route is available for unregistered users too